presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/ColumnMapping.java

```diff
@@ -71,7 +102,7 @@ public static ColumnMapping doubleMapping(
 
     public static ColumnMapping sliceMapping(Type prestoType, SliceReadFunction readFunction, SliceWriteFunction writeFunction)
     {
-        return sliceMapping(prestoType, readFunction, writeFunction, FULL_PUSHDOWN);
+        return sliceMapping(prestoType, readFunction, writeFunction, fullPushDown());
```

Comment:
Actually why did you changed constants into factory methods?
They are still constant anyway.

If just for the code style reasons, i would suggest separating to a prep commit (or reverting)